net/frr7/Makefile

```diff
@@ -34,7 +34,7 @@ INFO=		frr
 
 OPTIONS_DEFINE=		CONFIGROLLBACKS FPM DATACENTER LDPD PAM SHELLACCESS \
 			TCPSOCKETS VTYSH MANPAGES MULTIPATH TCMALLOC RPKI SNMP
-OPTIONS_DEFAULT=	VTYSH RPKI FPM MANPAGES
+OPTIONS_DEFAULT=	VTYSH RPKI FPM MANPAGES SNMP
```

Comment:
We don't change items from port that can be changed with environment variables.  Please remove this change and submit a separate PR to pfSense main repository enabling this option on make.conf at
https://github.com/pfsense/pfsense/blob/master/tools/conf/pfPorts/make.conf#L227

Reply:
Sure, no problem.

Reply:
PR: https://github.com/pfsense/pfsense/pull/4359